charts/opensearch-cluster/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: opensearch-cluster
+description: The OpenSearch Cluster Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
```

Comment:
I think the version should align with the version of the operator.

Reply:
Have put it to the current operator version.